sunpy/map/mapbase.py

```diff
@@ -1416,7 +1416,7 @@ def superpixel(self, dimensions: u.pixel, offset: u.pixel=(0, 0)*u.pixel, func=n
 # #### Visualization #### #
 
     @u.quantity_input
-    def draw_grid(self, axes=None, grid_spacing: u.deg = 15*u.deg, **kwargs):
+    def draw_grid(self, axes=None, annotation=False, grid_spacing: u.deg = 15*u.deg, **kwargs):
```

Comment:
```suggestion
    def draw_grid(self, axes=None, annotation=True, grid_spacing: u.deg = 15*u.deg, **kwargs):
```
I think we want this to default to be on.